src/components/accession2/ViewPhotosModal.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useRef } from 'react';
+import strings from 'src/strings';
+import Button from '../common/button/Button';
+import DialogBox from '../common/DialogBox/DialogBox';
+import Carousel from 'react-multi-carousel';
+import 'react-multi-carousel/lib/styles.css';
+import { Box } from '@mui/material';
+
+export interface ViewPhotosDialogProps {
+  open: boolean;
+  onClose: () => void;
+  photosNames: string[];
+  accessionId: string;
+  selectedSlide: number;
+}
+
+export default function ViewPhotosDialog(props: ViewPhotosDialogProps): JSX.Element {
+  const { onClose, open, photosNames, accessionId, selectedSlide } = props;
+
+  const myCarousel = useRef<Carousel>(null);
+  const responsive = {
+    mobile: {
+      breakpoint: { max: 4000, min: 0 },
+      items: 1,
+    },
+  };
+
+  useEffect(() => {
+    if (myCarousel.current) {
+      myCarousel.current.goToSlide(selectedSlide);
+    }
+  }, [selectedSlide, open]);
+
+  return (
+    <DialogBox
+      onClose={onClose}
+      open={open}
+      title={strings.PHOTOS}
+      size='large'
+      middleButtons={[
+        <Button
+          label={strings.PREVIOUS_ARROW}
+          priority='secondary'
+          onClick={() => {
+            if (myCarousel.current) {
+              const prevSlide = myCarousel.current.state.currentSlide - 1;
```

Comment:
guessing the left and right buttons are disabled when they are at the beginning and last? i don't have pictures in staging so don't know how to test vercel.

Reply:
They are not disabled, but they don't do anything if you click them

Reply:
Ah ok, we should disable if currentSlide is 0 and same for the other one if currentSlide is max?